_posts/2024-09-16-introducing-opensearch-2-17.md

```diff
@@ -0,0 +1,109 @@
+
+**Accelerate search performance with new techniques for computationally intensive queries**
+
+As part of our ongoing effort to improve search performance, we are introducing a new set of experimental features to improve performance of computationally expensive queries. Our approximation framework brings new techniques to short-circuit long-running queries by only scoring relevant documents in the query. The first queries to take advantage of this framework in version 2.17 are top-level range queries with no other clauses. These queries show benchmark results of up to 50x faster query running times with this optimization. To enable these new experimental features, set `opensearch.experimental.feature.approximate_point_range_query.enabled` to `true` in `opensearch.yml`.
+
+
+## Observability and log analytics
+
+This release provides an expansion of OpenSearch’s observability and log analytics capabilities.
+
+**Enhance trace analysis with a new custom trace source for advanced filtering and cross-cluster support**
+
+OpenSearch 2.17 introduces a new custom trace source as an experimental feature. This new trace source is based on the OpenTelemetry schema and includes a redesigned overview page for traces and services, which offers advanced filtering capabilities on trace metadata and optimized performance. The new experimental custom trace source will also provide flexibility for wildcards in index names and support for cross-cluster trace indexes.
+
+**Unlock enhanced anomaly detection with flexible data sources, advanced imputation methods, and domain-specific rules**
+
+With OpenSearch 2.17 release, we have added the following enhancements to the Anomaly Detection plugin:
+
+* **Support for remote and multiple data sources**: The Anomaly Detection Dashboards now support remote index selection, allowing you to select indexes from remote clusters in addition to local ones. You can choose any number of remote and local indexes, providing greater flexibility in data analysis.
```

Comment:
Should "dashboards" be lowercase? I would delete "providing greater flexibility in data analysis" as unnecessary.

Reply:
Reworded.